### PR TITLE
Formstack Forms v1

### PR DESCRIPTION
**Pull request by bbillstrFS**
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by bbillstrFS** [comment]
Hey @dougelliott - do you need anything from us to assist with the code review here? Looking forward to getting Forms submitted in ISV studio!

File: certified-connectors/Formstack Forms/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,243 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Formstack",
```
Comment (dougelliott) [inline certified-connectors/Formstack Forms/apiDefinition.swagger.json:4]: Just to double-check, should the title be `Formstack` or `Formstack Forms`?

Comment (bbillstrFS) [reply certified-connectors/Formstack Forms/apiDefinition.swagger.json:4]: @dougelliott Nice catch - I will change and update the PR with a new commit.  Anything else we see being an issue?

**Review by sriyen-msft — DISMISSED** [review]

File: certified-connectors/Formstack Forms/apiDefinition.swagger.json (hunk 1 above)
Comment (dougelliott) [reply certified-connectors/Formstack Forms/apiDefinition.swagger.json:4]: @bbillstrFS With Srikanth's approval coming in, that should be it :)

Comment (bbillstrFS) [reply certified-connectors/Formstack Forms/apiDefinition.swagger.json:4]: @dougelliott Confirmed with Gabe that we want Formstack Forms here - is the title in swagger necessary, or does the name we specify when we submit to ISV Studio the main driver for connector name?

Comment (dougelliott) [reply certified-connectors/Formstack Forms/apiDefinition.swagger.json:4]: @bbillstrFS It's actually the other way around, the title in the swagger is what matters most, as that is what will be used across the product. The name in ISV Studio should ideally match, but it has no bearing on what ends up getting deployed. So if you'd like Formstack Forms, go ahead and change this field as such and you'll be good to go.

Comment (bbillstrFS) [reply certified-connectors/Formstack Forms/apiDefinition.swagger.json]: @dougelliott Fantastic - thanks so much for the quick replies here!  This change has been made and latest commit should be tied to this PR.

**Review by dougelliott — APPROVED** [review]

File: certified-connectors/Formstack Forms/apiDefinition.swagger.json (hunk 1 above)
Comment (dougelliott) [reply certified-connectors/Formstack Forms/apiDefinition.swagger.json]: @bbillstrFS Awesome, approved. Go ahead and submit in ISV Studio and we'll get things started.

**Pull request merged**